### PR TITLE
Parametrizations fix

### PR DESCRIPTION
**Pull request by Lezcano**
This PR fixes a couple formatting errors that the tutorial on parametrizations has. In particular, I forgot to put a `super().__init__()` in an `nn.Module`  and a section was being rendered as a comment.

I also took the chance point out explicitly in the tutorial that parametrizations are just Python properties, and I made some minor stylistic changes.

@albanD

### CODE REVIEW

File: intermediate_source/parametrizations.py [hunk 1]
```diff
@@ -132,30 +132,41 @@ def forward(self, X):
...
 #
-# 1) It has a new ``module.parametrizations`` attribute
+# 2) It has a new ``module.parametrizations`` attribute
 #
-# 2) The weight has been moved to ``module.parametrizations.weight.original``
+# 3) The unparametrized weight has been moved to ``module.parametrizations.weight.original``
 #
 # |
-# We may observe this first change by printing the module
+# After parametrizing ``weight``, we see that, even though calling ``layer.weight`` returns
```
Comment (albanD) [inline intermediate_source/parametrizations.py:144]: I feel like this is a bit ambiguous.
And I don't think you want to go into the details of showing the property with `__class__` below.

Mentioning that `layer.weight` is now a property that contains the parametrized value of the weight. Sounds simpler.

File: intermediate_source/parametrizations.py [hunk 2]
```diff
@@ -132,30 +132,41 @@ def forward(self, X):
...
+# a tensor, ``layer.weight`` is not a tensor itself, but a property:
 layer = nn.Linear(3, 3)
-print("Unparametrized:")
-print(layer)
+parametrization = Symmetric()
+parametrize.register_parametrization(layer, "weight", parametrization)
+print(layer.__class__.weight)
+
+###############################################################################
+# This property computes ``parametrization(weight)`` every time we request ``layer.weight``
```
Comment (albanD) [inline intermediate_source/parametrizations.py:152]: This is a good description I think.

**Review by albanD — COMMENTED** [review]
Looks mostly good.
Did you had a chance to run the script locally to make sure it all works well?

**Comment by Lezcano** [comment]
I just corrected that part making it less technical, and adding a link to the definition of properties in the standard library.

And yes, I run the script. Now everything should be fine.

**Comment by holly1238** [comment]
Thanks! :)

**Pull request merged**